File: isRotated/isRotated.py
import sys

def isRotated(s1, s2):
	if (len(s1) == len(s2)):
		for r in range(len(s1)):
			for i in range(len(s1)):
				if s1[i] != s2[(i + r) % len(s1)]: break
				elif i == len(s1) - 1: return 1
	return 0

if len(sys.argv) < 3:
	print("Requires 2 strings as arguments.")
	exit(1)

print("true" if isRotated(sys.argv[1], sys.argv[2]) else "false")
print("\n")

File: isRotated/isRotated.java
class isRotated{
	static boolean checkRotation(String s1, String s2) {
		if (s1.length() == s2.length()) {
			for (int r = 0; r < s1.length(); r++) {
				for (int i = 0; i < s1.length(); i++) {
					if (s1.charAt(i) != s2.charAt((i + r) % s1.length())) break;
					else if (i == s1.length() - 1) return true;
				}
			}
		}

		return false;
	}

	public static void main(String[] args) {
		if (args.length < 2) {
			System.out.println("Requires 2 strings as arguments.");
			System.exit(1);
		}

		System.out.println(checkRotation(args[0], args[1]) ? "true" : "false");
	}
}